docs/reference/cli-reference/cli-ref-restore.md

```diff
@@ -50,6 +50,25 @@ where `<projectPath>` specifies the location of a solution or a `packages.config
 
 Also see [Environment variables](cli-ref-environment-variables.md)
 
+## Lock file options
+With version 5.3.1, `nuget.exe` supports [lock file as an optional feature](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#locking-dependencies). The lock file can be enabled by setting the MSBuild property `RestorePackagesWithLockFile` for your project.
```

Comment:
Perhaps this should be a relative link, like the other links in this file. The file name would then be case-sensitive on GitHub, though.

```suggestion
With version 5.3.1, `nuget.exe` supports [lock file as an optional feature](../../consume-packages/Package-References-in-Project-Files.md#locking-dependencies). The lock file can be enabled by setting the MSBuild property `RestorePackagesWithLockFile` for your project.
```

Reply:
The guide for adding links to docs is here: https://docs.microsoft.com/en-us/contribute/how-to-write-links.

Also, nuget.exe supported lock files since 5.3.0. We no longer have a download link for that version on nuget.org/downloads, but that's when the lock file features were first added.